### ISSUE TITLE
模型中step定义

### ISSUE THREAD

**Issue by sevenandseven** [FlagOpen/FlagEmbedding#821]
你好，我想请问以下训练过程中的step和global step是如何定义的；
我目前有9106条数据，用了一张卡训练，每张卡上的batchsize是64，梯度累计步数是128：
global_step如果是：9106/128 = 71
step是：（9106/128）* epoch 

但是终端上显示的是：（我尝试用tensorboard进行绘图时，发现他并没有保存图像）
andb:                    train/epoch ▁▃▄▆██
wandb:              train/global_step ▁▃▅▆██
wandb:                train/grad_norm █▆▃▁▁
wandb:            train/learning_rate █▆▅▃▁
wandb:                     train/loss █▇▂▃▁
wandb:               train/total_flos ▁
wandb:               train/train_loss ▁
wandb:            train/train_runtime ▁
wandb: train/train_samples_per_second ▁
wandb:   train/train_steps_per_second ▁
wandb: 
wandb: Run summary:
wandb:                    train/epoch 4.51
wandb:              train/global_step 5
wandb:                train/grad_norm 2.49198
wandb:            train/learning_rate 0.0
wandb:                     train/loss 0.7259
wandb:               train/total_flos 0.0
wandb:               train/train_loss 0.75921
wandb:            train/train_runtime 137.7201
wandb: train/train_samples_per_second 330.598
wandb:   train/train_steps_per_second 0.036

为什么他的global_step 5？请问这个地方应该怎么计算？

**Comment by staoxiao** [comment]
Global step refers to the number of times the model has been updated: `global_step = 9106/(64*128) * 5 = 5`

**Comment by sevenandseven** [comment]
> Global step refers to the number of times the model has been updated: `global_step = 9106/(64*128) * 5 = 5`. Besides, the accumulation step is too large (128), and I think setting it to 1 is enough.

ok,  The setting of gradient accumulation steps is related to the batch size. It is not necessary to be larger than the batch size. The step how to calculated?

**Comment by staoxiao** [comment]
> The step how to calculated?

I think step is equal to global_step
